### DISCUSSION TITLE
[done] Создать параметр sleep

### DISCUSSION THREAD

**Discussion by an220022** [totumonline/totum-ru-issues-and-discussions#22] [Есть идея, а что если...]
Дважды столкнулся с проблемой ограничения количества запросов в секунду на API стороннего сервиса (Например: не более 50 запросов в сек). Пришлось обходить ограничения через расписание крона, что не очень удобно. Предлагаю создать параметр sleep: (время в секундах) для while.

**Comment by ttm-alexeypolunin** [comment]
Мы думаем, что это будет функция `sleep` с единственным параметром `sec:`.  Для разрабов вроде несложно — включим в релиз на php8.

**Comment by ttm-alexeypolunin** [comment]
Добавлена в 2.* — https://docs.totum.online/functions#fn-sleep
